go/vt/vtorc/logic/topology_recovery.go

```diff
@@ -775,17 +775,16 @@ func CheckAndRecover() {
 		log.Error(err)
 		return
 	}
+
 	// intentionally iterating entries in random order
 	for _, j := range rand.Perm(len(replicationAnalysis)) {
 		analysisEntry := replicationAnalysis[j]
 
 		go func() {
-			err = executeCheckAndRecoverFunction(analysisEntry)
```

Comment:
This looked like a bug to me so changed the scoping here.

Reply:
Oh yes! Good catch.